torch/csrc/autograd/variable.h

```diff
@@ -274,6 +310,18 @@ struct TORCH_API AutogradMeta : public c10::AutogradMetaInterface {
 ///
 /// Differentiable Views
 /// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/// This class allows to track both forward and backward AD differentiable views.
+/// These views can have different base as non-differentiable view for forward
+/// and backward mode AD are not the same.
```

Comment:
This would be clearer with an example. And there are a few places where you obliquely refer to the fact that the views could be different, which would do well referencing here